gatsby-config.ts

```diff
@@ -1,11 +1,16 @@
 /* eslint-disable no-restricted-globals */
+import * as dotenv from 'dotenv';
 import type { GatsbyConfig } from 'gatsby';
 
+dotenv.config({
+  path: `.env`,
+});
```

Comment:
`.env` is the default path.
```suggestion
dotenv.config();
```